spec/spec_helper.rb

```diff
@@ -32,4 +35,7 @@
   end
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
+
+  OmniAuth.config.test_mode = true
+  WebMock.allow_net_connect!
```

Comment:
I had to change this to allow for selenium chrome driver to work properly and get Travis CI to pass.

Reply:
good catch